lib/helper/Nightmare.js

```diff
@@ -835,6 +836,31 @@ class Nightmare extends Helper {
     return array.length === 1 ? array[0] : array;
   }
 
+  /**
+   * {{> grabCssPropertyFrom }}
+   */
+  async grabCssPropertyFrom(locator, cssProperty) {
+    locator = new Locator(locator, 'css');
+    const els = await this.browser.findElements(locator.toStrict());
+    const array = [];
+
+    const getCssPropForElement = async (el, prop) => {
+      return (await this.browser.evaluate((el) => {
+        return window.getComputedStyle(window.codeceptjs.fetchElement(el));
+      }, el))[toCamelCase(prop)];
+    };
+
+    for (let index = 0; index < els.length; index++) {
+      const el = els[index];
+      assertElementExists(el, locator);
+      const cssValue = await getCssPropForElement(el, cssProperty);
+      array.push(cssValue);
```

Comment:
may we try using `forEach`?

Reply:
Good point @PeterNgTr  :) 
Can we consider `for of` ? I think it is more readable. Your view?

Reply:
should be good :)